Week-9/todo-list/src/components/ToDoItem.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useId } from "react";
+
+const ToDoItem = (props) => {
+  const [checked, setChecked] = useState(false);
+  const uuid = useId();
+
+  const handleChange = () => setChecked(!checked);
+
+  return (
+    <div className="todo-item" id={uuid}>
+      <input
+        type="checkbox"
+        checked={checked === true ? "checked" : ""}
```

Comment:
`checked={checked ? "checked" : ""}` Will this work? I dont think we need to explicitly check if its true .